src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1SpaceBookingEntity.kt

```diff
@@ -112,6 +114,51 @@ interface Cas1SpaceBookingRepository : JpaRepository<Cas1SpaceBookingEntity, UUI
     pageable: Pageable?,
   ): Page<Cas1SpaceBookingSearchResult>
 
+  @Query(
+    value = """
+      SELECT 
+      Cast(b.id as varchar),
+      b.crn as crn,
+      b.canonical_arrival_date as canonicalArrivalDate,
+      b.canonical_departure_date as canonicalDepartureDate,
+      apa.risk_ratings -> 'tier' -> 'value' ->> 'level' as tier,
+      apa.name as personName,
+      apa.release_type as releaseType,
+      ( 
+        SELECT ARRAY_AGG (characteristics.property_name)
+        FROM cas1_space_bookings_criteria sbc
+        LEFT OUTER JOIN characteristics ON characteristics.id = sbc.characteristic_id
+        WHERE sbc.space_booking_id = b.id 
+        GROUP by sbc.space_booking_id
+      ) AS characteristics
+      FROM cas1_space_bookings b
+      INNER JOIN approved_premises_applications apa 
```

Comment:
bookings may be linked to offline applications instead of approved_premises_applications. given this, this should be a left join and an additional left join on offline_applications is required

to get the person's name can use

```
      CASE 
        WHEN apa.id IS NOT NULL THEN apa.name
        ELSE offline_app.name
      END as personName
```

release type is unknown for offline applications so will be null

would be good to have an integration test that included a booking linked to an offline applications. existing tests for the space booking search endpoint should be doing this already so can use that as a guide